regex.yaml

```diff
@@ -1164,13 +1164,13 @@ regular_expresions:
     # Disable in winpeas because of RegexDoS (for some reason)
     - name: Basic Auth
       regex: >
-        //(.+):(.+)@
+        //(?:.+):(?:.+)@
       example: >
         //UserName237.e4r3%:Compl3x&/Password1763@domain.com
 
     - name: PHP Passwords
       regex: >
-        (pwd|passwd|password|PASSWD|PASSWORD|dbuser|dbpass|pass').*[=:].+|define ?\('(\w*pass|\w*pwd|\w*user|\w*datab)
+        (?:pwd|passwd|password|PASSWD|PASSWORD|dbuser|dbpass|pass').*[=:].+|define ?\(?:'(?:\w*pass|\w*pwd|\w*user|\w*datab)
```

Comment:
If you only merge this commit and not the other one please edit this pattern as I made a mistake
I didn't notice that the parenthesis was escaped.
`(?:pwd|passwd|password|PASSWD|PASSWORD|dbuser|dbpass|pass').*[=:].+|define ?\('(?:\w*pass|\w*pwd|\w*user|\w*datab)`

If you merge all my commits, that's fine as I fix it in the other one.